docs/build_developer_images.md

```diff
@@ -28,15 +28,15 @@ OS_RELEASE=n hack/build-go.sh
 ```
 cp _output/local/bin/linux/amd64/openshift images/origin/bin/
 
-docker build -t openshift/origin images/origin
+hack/build-local-images.py origin
 ```
 
 ### Build dependent images 
 
 #### Router Image
 
 ```
-docker build -t openshift/origin-haproxy-router images/router/haproxy/
+hack/build-local-images.py haproxy-router
 ```
 
 ## Deploying Images
```

Comment:
I think it's worth mentioning that in a dind environment you can just say hack/dind-cluster.sh copy-image origin-haproxy-router

And it will copy it to the inner pages.